requirements.txt

```diff
@@ -1,6 +1,7 @@
 Flask
 gunicorn
 Flask-Cache
```

Comment:
This should use logic to check for version. We don't want both, we just want the applicable one.

```+if platform.python_version()[:3] == '2.6':
+    from flask.ext.cache import Cache
+else:
+    from flask_caching import Cache```

Reply:
Is it possible to have logic in a .txt file? I have logic in the __init__.py for tinker where cache is initialized.

Reply:
oh good point. I think thats fine then